merge-v1/schema.graphql

```diff
@@ -25,30 +25,38 @@ An account of users holding an NFT within merge. collection.
 type User @entity {
   "The holder's address of NFT in collection"
   id: ID!
-  "Unique tokenID for a respective NFT"
-  tokenID: BigInt!
   "Details about NFTs being held by this User"
-  massNFTs: [NFTs!]! @derivedFrom(field: "owner") #pak: allows one to query info about NFTs belonging to this user
+  massNFTs: [NFT!] @derivedFrom(field: "owner") #pak: allows one to query info about NFTs belonging to this user
   "Whether this User can carry more than one mass NFT"
-  whitelist: Boolean! #pak: These are central marketplaces typically like Nifty Gateway
+  whitelist: Boolean #pak: These are central marketplaces typically like Nifty Gateway
 }
 
 """
 Individual NFTs within this collection.
 """
-type NFTs @entity {
+type NFT @entity {
   "Unique tokenID"
   id: ID!
   "Whether the NFT has been merged, and therefore deleted"
-  merged: Boolean! #pak
+  merged: Boolean #pak
   "Current owner unless merged where it would be address(0)"
   owner: User!
-  "Size of mass for a specific NFT"
+  "Core metric: used for each NFT in smart contracts to get derived metrics"
```

Comment:
I'll move it to a comment. Since the docstring is for those that may not even know what is going on in the smart contracts, moving it to a comment makes sense.